### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by diskurse** [Audrey-Randall/stalkerware#1]
Just came across this repo and thought I'd reach out as perhaps we are doing similar research? https://github.com/diskurse/android-stalkerware

**Comment by Audrey-Randall** [comment]
Hi!

We are indeed doing similar work! My name is Audrey, I'm a PhD student at
the University of California San Diego. This repo is from a class project
that I would like to turn into something more substantial. There are also
two other collaborators on it at the moment, Yu-Wun and Hugh. I've only
just skimmed your repo but I did notice that you got better information
from TheTruthSpy's network capture than we did. How are you tapping the
traffic? We just turned our laptops into mobile hotspots and had the phones
connect over WiFi.

May I ask who you're working for/with, or if this just an independent
project of yours? I saw that you reported one of these spyware companies to
Motherboard for exposing user data. (Thank you!) Are you working in any
particular direction, or just gathering information and seeing what comes
up?

Also, if you haven't found them yet, a third group doing research in this
area is Dr. Thomas Ristenpart's team from Cornell. (
https://www.ipvtechresearch.org/post/publications/) They have a partnership
with a New York domestic violence shelter and are specifically focusing on
tech abuse in intimate partner violence, which includes stalkerware but
also extends a long way beyond it. They have a really great grasp of the
non-technical challenges associated with protecting victims of IPV.

Thank you for reaching out! Looking forward to hearing about your work.
-Audrey Randall
